datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -216,6 +216,70 @@ SELECT approx_median(a) FROM median_f64_nan
 ----
 NaN
 
+# median_multi
```

Comment:
I ported the tests to sqllogictest as much of the rest of the aggregate tests had been ported too